pkg/validation/config.go

```diff
@@ -402,6 +407,21 @@ func validatePromotionConfiguration(fieldRoot string, input api.PromotionConfigu
 	if len(input.Name) != 0 && len(input.Tag) != 0 {
 		validationErrors = append(validationErrors, fmt.Errorf("%s: both name and tag defined", fieldRoot))
 	}
+
+	if promotesOfficialImages {
+		var latestRelease bool
+		if releaseTagConfiguration != nil {
+			latestRelease = true
+		}
+		if releases != nil {
+			if _, ok := releases["latest"]; ok {
```

Comment:
Fun fact: map indexing has an implicit `nil` check in Go.  This can all be just:

```go
if _, ok := releases["latest"]; !ok && releaseTagConfiguration == nil {
```